app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -721,42 +748,48 @@ class InputInteractionViewTestActivityTest {
 
   @Test
   fun testRatioInputInteractionView_withNoInputText_hasCorrectPendingAnswerType() {
-    val activityScenario = ActivityScenario.launch(
+    launch(
```

Comment:
@anandwana001 Tried implementing suggested changes, but it is still failing on robolectric